types/src/vm_error.rs

```diff
@@ -399,6 +399,8 @@ pub enum StatusCode {
     STORAGE_ERROR = 2008,
     INTERNAL_TYPE_ERROR = 2009,
     EVENT_KEY_MISMATCH = 2010,
+    // The VM failed to start. This can be caused by e.g. missing the account module.
+    VM_FAILED_TO_START = 2011,
```

Comment:
When are we expecting to see this error code? IIUC we will return an failure::Error when the VM fails to initialize. Is the idea that we are going to convert this error code into failure::Error?

Reply:
Ah, this will be coming in the upcoming PR. I'll remove from this one. A VMStatus does implement `Error` so there's no conversion needed.